src/pages/AppDetail.svelte

```diff
@@ -60,7 +63,11 @@
     {/each}
   </ul>
   <h2>Metrics</h2>
-  <FilterInput onChangeText={filterMetrics} />
+  {#if app.metrics.length === 0}
+    <p>Currently, there are no metrics available for {app.name}</p>
+  {:else}
+    <FilterInput onChangeText={filterMetrics} />
+  {/if}
```

Comment:
I would probably put the content below in the else (I know it doesn't make a visual difference, but it makes more sense this way)